Changelog.md

```diff
@@ -1,3 +1,12 @@
+Unreleased
+-------------------------
+
+* [Bug fix] Add a check to the `add_user_forgeign_key` migration
```

Comment:
Typo, should be "foreign" (same in the commit message and PR description).

Reply:
Thanks, fixed.